RELEASE.md

```diff
@@ -0,0 +1,17 @@
+# Release process
+
+The release process for CNAO consist on a pair of manual steps:
+1. Create release PR with github action [Create Version PR](https://github.com/kubevirt/cluster-network-addons-operator/actions/workflows/prepare-version.yaml), 
+   it have a pair of fields `versionLevel` with means the semVer version part 
+   to bump and `baseBranch` the branch to relase from. To run it:
+   - From the gh UI passing the `versionLevel` or `baseBranch`
+   - With the gh cli like `gh workflow run prepare-version.yaml -s versionLevel=... -s baseBranch=...`
+2. Merge the version PR when continous integration is fine. 
+
+After the version PR is merged a prow job will [release-cluster-network-addons-operator](https://github.com/kubevirt/project-infra/blob/main/github/ci/prow-deploy/files/jobs/kubevirt/cluster-network-addons-operator/cluster-network-addons-operator-postsubmits.yaml#L2-L28) this will call the `make release` target that do the following:
+
+1. Tag the code the version from the version PR.
+2. Generate release notes from the PRs release-notes field at description
```

Comment:
I miss the information on what are the release notes generated from. How can I mark a PR to be included there, how can I mark it not to get listed, how can I mark it as "Urgent Upgrade Note".